lib/client/db/profile.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package db contains methods for working with database connection profiles
```

Comment:
Sounds like `lib/client/db/profile` would be a more apt package location for working with profiles

Reply:
Can't move it there due to circular imports.

Reply:
I'm not sure which imports you mean here, but my suggestion was to pull the interfaces (`ConnectProfileFile` and `ConnectProfile`) to `lib/client/db` and implement support for profile handling for a specific database flavor in the corresponding `lib/client/db/profile/<profile>` or `lib/client/db/<flavor>/`.